### PR TITLE
New card implemented: Clone

### PR DESCRIPTION
**Pull request by brubraz**

### CODE REVIEW

File: DecisionQueue/DecisionQueueEffects.php [hunk 1]
```diff
@@ -223,9 +223,11 @@ function PlayerTargetedAbility($player, $card, $lastResult)
   }
 }
 
-function SpecificCardLogic($player, $card, $lastResult)
+function SpecificCardLogic($player, $paramter, $lastResult)
```
Comment (VJubert) [inline DecisionQueue/DecisionQueueEffects.php:226]: parameter

File: DecisionQueue/DecisionQueueEffects.php [hunk 2]
```diff
@@ -464,8 +466,8 @@ function SpecificCardLogic($player, $card, $lastResult)
       return $lastResult;
     case "L337":
       $target = $lastResult;
-      if($target == "PASS") {
-        $ally = new Ally("MYALLY-" . SearchAlliesForCard($player, "9552605383"), $player);
+      if($target == "PASS") {
```
Comment (VJubert) [inline DecisionQueue/DecisionQueueEffects.php:469]: wrong end space

File: CoreLogic.php [hunk 3]
```diff
@@ -2098,7 +2102,7 @@ function PlayAbility($cardID, $from, $resourcesPaid, $target = "-", $additionalC
     
   if($from == "PLAY" && IsAlly($cardID, $currentPlayer)) {
     $playAlly = new Ally("MYALLY-" . $index);
-    $abilityName = GetResolvedAbilityName($cardID, $from);
+    $abilityName = GetResolvedAbilityName($cardID, $from);
```
Comment (VJubert) [inline CoreLogic.php:2105]: wrong end space

Comment (brubraz) [reply CoreLogic.php]: Done, thanks!

File: DecisionQueue/DecisionQueueEffects.php (hunk 1 above)
Comment (brubraz) [reply DecisionQueue/DecisionQueueEffects.php]: Done, thanks!

**Review by OotTheMonk — APPROVED** [review]

**Pull request merged**